src/components/Authclient/CAuthclientEditorInfo.vue

```diff
@@ -454,6 +497,21 @@ export default {
   },
 
   methods: {
+
+    copyToClipboard () {
+      copy(this.cUrl)
+    },
+
+    generateSnippet () {
```

Comment:
How about if you put the entire curl command directly into the HTML template?

Reply:
Not sure what you mean, into a div, span?